src/Service.Host/package.json

```diff
@@ -31,6 +31,7 @@
     "author": "",
     "license": "ISC",
     "devDependencies": {
+        "autoprefixer": "6.3.7",
```

Comment:
should I bring all these package.json changes into the template?

Reply:
Yeah, this one was breaking the builds after updating.